### PR TITLE
Change Target Panel Editor styles

### PR DESCRIPTION
**Pull request by ferhoyos**

### CODE REVIEW

File: frontend/src/pages/Mesh/target/TargetPanelEditor.tsx [hunk 1]
```diff
@@ -1,36 +1,60 @@
...
+  showPrintMargin: false
+};
+
+const aceEditorStyle = kialiStyle({
+  marginTop: '0.5rem',
+  backgroundColor: PFColors.BackgroundColor100,
+  $nest: {
+    '& .ace_gutter': {
+      backgroundColor: PFColors.BackgroundColor100,
+      borderRight: 0
```
Comment (josunect) [inline frontend/src/pages/Mesh/target/TargetPanelEditor.tsx:30]:
```
,
      left: "-10px !important",
    },
```

What do you think about reducing a little bit the left margin?

File: frontend/src/pages/Mesh/target/TargetPanelEditor.tsx [hunk 2]
```diff
@@ -1,36 +1,60 @@
...
+    },
+    '& .ace_cursor': {
+      opacity: 0
+    }
```
Comment (josunect) [inline frontend/src/pages/Mesh/target/TargetPanelEditor.tsx:34]:
```
,
    '& .ace_scroller': {
      left: "25px !important"
    }
```

**Review by josunect — COMMENTED** [review]
Thanks, @ferhoyos, looks good, i've just added one comment.

File: frontend/src/pages/Mesh/target/TargetPanelEditor.tsx (hunk 1 above)
Comment (ferhoyos) [reply frontend/src/pages/Mesh/target/TargetPanelEditor.tsx]: Agreed, I have moved the editor a bit to the left as suggested.

**Review by josunect — APPROVED** [review]

**Pull request merged**